testfiles/TF14/14.1-1-fail-1.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+
+   <head>
+      <title>TC14.1-1-fail-1 Image Hyperlink Example </title>
+   </head>
+	
+   <body>

```

Comment:
add code snippet div

Reply:
Not sure what you are asking for? add code snippet div?